### PR TITLE
IGNITE-15363 Update Ignite Spring Data Dependency Snippet

### PR DESCRIPTION
**Pull request by Nikita-tech-writer**
Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [ ] There is a single JIRA ticket related to the pull request. 
- [ ] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: docs/_docs/extensions-and-integrations/spring/spring-data.adoc [hunk 1]
```diff
@@ -33,31 +33,7 @@ tab:pom.xml[]
 <dependency>
     <groupId>org.apache.ignite</groupId>
     <artifactId>ignite-spring-data-2.2-ext</artifactId>
-    <version>${ignite-spring-data-ext.version}</version>
```
Comment (alamar) [inline docs/_docs/extensions-and-integrations/spring/spring-data.adoc:36]: Also in this doc:

> Replace `${ignite-spring-data-ext.version}`, `${spring.data.version}`, and

Is it still relevant?

**Comment by ololo3000** [comment]
All spring data extension dependencies are marked as provided based on discussion - [1]. Current documentation shows all dependencies required by spring data extension. Why should we remove them? 

Also the current documentation already shows the correct versions available in the table. It helps to match version of extension with supported versions of Ignite.

[1] - https://lists.apache.org/thread.html/r0c97640b9c5f11a02fb6d5ea39ffb26edc0fd8215a479fa46e5d54fa%40%3Cdev.ignite.apache.org%3E

**Comment by ololo3000** [comment]
> @ololo3000 Can you please confirm then that the docs are in up-to-date state and this PR can be closed?
@Nikita-tech-writer From my point of view - yes, they are. At least i see no reasons for the current changes.

**Comment by NSAmelchev** [comment]
Close the PR due to the extensions section was added to the site.

**Pull request closed without merging**